arangograph/notebooks.md

```diff
@@ -95,6 +95,8 @@ Single line commands have `%` prefix and multi-line commands have `%%` prefix.
 - `%whichDatabase` - returns the database name you are connected to.
 - `%createDatabase databaseName` - creates a database.
 - `%selectDatabase databaseName` - selects a database as the current database.
+- `%useDatabase databasename` - uses a database as the current database; shares
+  the same function with `%selectDatabase`.
```

Comment:
In other words, it is an alias for `%selectDatabase`?

Reply:
yes